_posts/2022-11-20-jump_dev_2023.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: page
+title:  "JuMP-dev 2023"
+date:   2022-11-20
+categories: [jump-dev]
+permalink: /meetings/jumpdev2023/
+---
+
+We are pleased to announce that JuMP-dev 2023 will be co-located with
+[JuliaCon 2023](https://juliacon.org/2023)!
+
+This year, JuliaCon is being held 24-29 July, 2023 in Cambridge, Massachusetts.
+You can find more information on the [conference website](https://juliacon.org/2023/).
+
+## Outline
+
+The purpose of JuMP-dev is to bring together students, researchers, and
+practitioners with interests in the methodological, algorithmic, and software aspects of
+[JuMP](https://github.com/jump-dev/JuMP.jl) and related packages. In particular,
+we invite new contributors and those who have not met the core development team.
+
+## How do I attend?
+
+To attend JuMP-dev, you must purchase a ticket to JuliaCon 2023. Ticket sales
+will open in early 2023.
+
+All participants will uphold the [JuliaCon Code of Conduct](https://juliacon.org/2023/coc/).
+
+## How do I give a talk?
+
+**To give a talk, [submit a proposal to JuliaCon](https://pretalx.com/juliacon2023/cfp)**
+**For "Session type," choose "Talk" or "Lightning talk." For "Track," choose**
+**"JuMP."**
+
+Similarly to [previous](/meetings/mit2017) [iterations](/meetings/bordeaux2018)
+[of](/meetings/santiago2019) [the](/meetings/juliacon2021) [workshop](/meetings/juliacon2022),
+we invite participants to present work related to JuMP. In particular, we
+especially seek talks related to:
+
+- JuMP core development ([MathOptInterface](https://github.com/JuliaOpt/MathOptInterface.jl), JuMP 1.0)
+- Mathematical optimization solvers written in Julia
+- Automatic differentiation in Julia
+- Julia interfaces to mathematical optimization solvers
+- JuMP extensions (stochastic programming, robust optimization, multiobjective optimization, ...)
+- Software that uses JuMP
+- Developer tools for JuMP
+- Commercial use of JuMP
+- Uses of JuMP in teaching
+
+Speakers are encouraged to highlight challenges and lessons learned, not only
+successes. Talks should be aimed at a general audience, but familiarity with
+JuMP/Julia can be assumed.
+
+See the previous workshops for examples of accepted talks: [2017](/meetings/mit2017/),
+[2018](/meetings/bordeaux2018/), [2019](/meetings/santiago2019),
+[2021](/meetings/juliacon2021), and [2022](/meetings/juliacon2022).
+
+## Programme committee
+
+ * Oscar Dowson
+ * Chris Coey (RAI)
+ * Joaquim Dias Garcia (PSR)
+ * Benoît Legat (MIT)
+ * Miles Lubin (HRT)
+ * Joshua Pulsipher (CMU)
+
+To contact the program committee, write to [jump-dev-2023@googlegroups.com](mailto:jump-dev-2023@googlegroups.com).
```

Comment:
Mailto links seem very 1990s? They don't really work unless you have standalone email client.